src/particle_system.cc

```diff
@@ -0,0 +1,67 @@
+/*************************************************************************
```

Comment:
You can remove this file, it is not needed anymore. Everything is in .hh, because of templates.